### DISCUSSION TITLE
Can submit a prompt?

### DISCUSSION THREAD

**Discussion by cms168** [logancyang/obsidian-copilot#730] [General]
Copilot has been great thus far, but for some reason, tonight the prompt window in the sidebar allows me to enter the text, but I cannot send it.  The send Icon can be clicked but nothing happens.  Just hitting enter on the keyboard does nothing.  I've restarted Obsidian, checked for updates, and still no luck.  Any suggestions?  Thank you!

**Comment by logancyang** [comment]
Is this related? https://github.com/logancyang/obsidian-copilot/issues/724

Please turn off all other plugins and try again, let me know if it still doesn't work.

**Comment by cms168** [comment]
Hi Logan, Thanks for that.  I turned all off except for Copilot.  That worked.  I then re-enabled one by one test each time.  Looks like Make.MD causes the submission box to not be able to submit.  I'm looking into it more, but if you have ideas, please let me know.  I'm not heavily attached to that plugin, but would be interested to see if it can coexist with Copilot.  Thanks again.
